easybuild/easyconfigs/h/HEALPix/HEALPix-3.82-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,59 @@
+easyblock = 'ConfigureMake'
+
+name = 'HEALPix'
+version = '3.82'
+
+homepage = 'http://healpix.sourceforge.net/'
+description = """Hierarchical Equal Area isoLatitude Pixelation of a sphere."""
+software_license = "LicenseGPLv2"
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['Healpix_3.82_2022Jul28.tar.gz']
+checksums = ['47629f057a2daf06fca3305db1c6950edb9e61bbe2d7ed4d98ff05809da2a127']
+
+unpack_options = '--strip-components=1'
+
+buildininstalldir = True
```

Comment:
we like to avoid building directly in the install dir where possible.

In this case, I would suggest considering the `MakeCp` easyblock (with `with_configure`) to run the configure, make steps, and then copy desired files/directories using `files_to_copy`

Reply:
thanks, `MakeCp` with `with_configure` and `files_to_copy` is indeed much cleaner.